huaweicloud/resource_huaweicloud_vpc_subnet.go

```diff
@@ -226,6 +243,12 @@ func resourceVpcSubnetV1Update(d *schema.ResourceData, meta interface{}) error {
 	//as name is mandatory while updating subnet
 	updateOpts.Name = d.Get("name").(string)
 
+	if d.HasChange("ipv6_enable") && !d.Get("ipv6_enable").(bool) {
+		return fmt.Errorf("Error updating ipv6_enable, IPv6 cannot disable")
+	} else {
+		enable := d.Get("ipv6_enable").(bool)
```

Comment:
the program will go here even if "ipv6_enable" has **not** changed.

Reply:
thanks, only if ipv6_enable has changed and the old value is false, enable can be set to updateopts.